bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -79,18 +79,36 @@ class SettingsDropdown extends Component {
   }
 
   componentWillMount() {
-    const { intl, isFullScreen, mountModal } = this.props;
+    this.createMenu(this.props);
```

Comment:
`this.props` doesn't need to be passed to `createMenu()`, you can just use `this.props` inside `createMenu()`